apps/useradmin/src/useradmin/views.py

```diff
@@ -198,7 +198,7 @@ def massage_groups_for_json(groups):
 
 def is_user_locked_out(username):
   attempts = AccessAttempt.objects.filter(username=username)
-  return any(attempt.failures_since_start >= FAILURE_LIMIT and LOCK_OUT_AT_FAILURE for attempt in attempts)
+  return any(attempt.failures_since_start >= settings.AXES_FAILURE_LIMIT and settings.AXES_LOCK_OUT_AT_FAILURE for attempt in attempts)
```

Comment:
Because of this error `ImportError: cannot import name 'FAILURE_LIMIT' from 'axes.decorators'
`

Reply:
after checking the code, find out that limits are shifted into the `conf` file.

Reply:
:+1: